eng/emitter-package.json

```diff
@@ -1,14 +1,14 @@
 {
```

Comment:
@weidongxu-microsoft could you add appropriate entries to the codeowners file for these emitter files?